### ISSUE TITLE
Minimize size and dependencies of flag icons

### ISSUE THREAD

**Issue by JustArchi** [JustArchiNET/ASF-ui#80]
```
flags should be added as external assets loaded from CDN
because you're not saving anything for the user in this case
as user is already downloading only images you're using
and not all flags from all countries
so the answer for "it's flags taking 300 KB"
is "why the hell do we include flags in the first place"

I'd just link to some general CDN with flags in small pngs
https://github.com/JustArchiNET/ArchiSteamFarm/wiki
look what I did on ASF wiki @Mole
https://github.com/hjnilsson/country-flags/tree/master/png100px
you don't need to use this one, there are dozens if not hundreds of different flag CDNs
which is why I don't know why instead of using them we're embedding those flags in the build
and then you also don't need to clean your build out of unused flags, simply because user will never download them off CDN if they're not used
```

**Comment by Aareksio** [comment]
Fixed in b02ad952a4e5c8c65f975819871efb009ba130dd, we're back at ~500kB

**Comment by Aareksio** [comment]
We need to know all flags that gonna be used at build time, otherwise webpack bundles all possible files.

This can be done in two ways:
- with `/* webpackInclude */` or `/* webpackExclude */` comments in dynamic import as documented [here](https://webpack.js.org/api/module-methods/#import-). Note the warning:

> Every module that could potentially be requested on an `import()` call is included. For example, ``import(`./locale/${language}.json`)`` will cause every `.json` file in the `./locale` directory to be bundled into the new chunk. At run time, when the variable `language` has been computed, any file like `english.json` or `german.json` will be available for consumption. Using the `webpackInclude` and `webpackExclude` options allows us to add regex patterns that reduce the files that webpack will bundle for this import.
- by creating utility helper that re-exports all needed flags:
```JS
export { default as pl } from 'svg-country-flags/png100px/pl.png';
export { default as de } from 'svg-country-flags/png100px/de.png';
...
```

I opted for the second one as it bundles all flags in single chunk. To avoid the need to manually update the helper, it's generated with `npm run generate-flags-helper`.

Currently the command needs to be executed whenever we add a new locale - we should consider hooking it to some action - ideally locale update. Is that possible, @JustArchi?
